.loco/config/mysql-common/my.cnf.php

```diff
@@ -38,12 +38,13 @@ function matchVer($pat) {return (bool) preg_match($pat, ver());}
 #innodb_lock_wait_timeout = 50
 
 <?php if (matchVer('/Ver 5.6/')) { printf("innodb_large_prefix = 1\n"); } ?>
+<?php if (matchVer('/Ver 5.6/')) { printf("sql_mode=\"STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION\"\n"); } ?>
```

Comment:
My experience is this isn't additive, i.e. this would overwrite the default not add to it. But I'm not familiar with the environment this runs under.

Reply:
by default MySQL 5.6 is only NO_ENGINE_SUBSITUTION this adds to it by including STRICT_TRANS_TABLES

Reply:
Ah ok got it. 👍